src/ansys/fluent/core/utils/fluent_version.py

```diff
@@ -20,3 +37,88 @@ def get_version_for_file_name(version: Optional[str] = None, session=None):
         version = get_version(session)
 
     return "".join(version.split(".")[0:2])
+
+
+@total_ordering
+class FluentVersion(Enum):
+    """An enumeration over supported Fluent versions.
+
+    Examples
+    --------
+    FluentVersion("23.2.0") == FluentVersion.v232
+
+    FluentVersion.v232.number == 232
+
+    FluentVersion.v232.awp_var == 'AWP_ROOT232'
+    """
+
+    v242 = "24.2.0"
+    v241 = "24.1.0"
+    v232 = "23.2.0"
+    v231 = "23.1.0"
+    v222 = "22.2.0"
+
+    @classmethod
+    def _missing_(cls, version):
+        if isinstance(version, (int, float, str)):
+            version = str(version)
+            if len(version) == 3:
+                version = version[:2] + "." + version[2:]
+            version += ".0"
+            for member in cls:
+                if version == member.value:
+                    return member
+        raise AnsysVersionNotFound(
+            f"The specified version '{version[:-2]}' is not supported."
+            + " Supported versions are: "
+            + ", ".join([member.value for member in cls][::-1])
+        )
+
+    @classmethod
+    def get_latest_installed(cls):
+        """Return the version member corresponding to the most recent, available ANSYS
+        installation.
+
+        Returns
+        -------
+        FluentVersion
+            FluentVersion member corresponding to the newest Fluent version.
+
+        Raises
+        ------
+        AnsysVersionNotFound
+            If an Ansys version cannot be found.
+        """
+        for member in cls:
+            if member.awp_var in os.environ:
+                return member
+
+        raise AnsysVersionNotFound(
+            "Verify the value of the 'AWP_ROOT' environment variable."
+        )
+
+    @classmethod
+    def current_release(cls):
+        """Return the version member of the current release.
+
+        Returns
+        -------
+        FluentVersion
+            FluentVersion member corresponding to the latest release.
+        """
+        return cls.v232
```

Comment:
To make maintaining this easier, so we don't have to dig into this file when a new Fluent release comes out, we could have a constant indicating the latest released Fluent version maybe in `src/ansys/fluent/core/_version.py` together with the PyFluent version?

Alternatively, a module constant defined at the top of this module with this information would also be better I think.

Then `FluentVersion.current_release()` parses this new variable and does not need to be separately maintained.

The doc build would ideally also reference this new constant, but this would be something to be discussed further and investigated (as doc build version specification is currently hardcoded in the YML files).

Reply:
Current release string added to _version.py